tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -27,6 +27,7 @@ Cce:
     BarycentricRationalSpanInterpolator:
       MinOrder: 10
       MaxOrder: 10
+  ExtractionRadius: 257.0
```

Comment:
Do I understand correctly that this should be set to `Auto` because the radius can be parsed from the file?

Reply:
It is true that it can be set to `Auto`, but I want to move away from the interface where numeric parameters are obtained from filenames (that was a SpEC thing...), so I'd prefer to show the example interface the way I think the code should be used most often. Supporting `Auto` is more to ease the transition for those that are still typically importing SpEC data.
Is that reasonable? Would you prefer the example have a filename that can't be parsed if it shows a numeric `ExtractionRadius`?

Reply:
I think it's fine as-is, thanks for clarifying :)